cmake/build_configurations/mysql_release.cmake

```diff
@@ -271,4 +276,20 @@ IF(UNIX)
       ENDIF()
     ENDIF()
   ENDIF()
+
+  # AIX flags
+  IF(CMAKE_SYSTEM_NAME MATCHES "AIX")
+    IF(CMAKE_COMPILER_IS_GNUCC)
+     SET(COMMON_C_FLAGS               "-fno-omit-frame-pointer -fno-strict-aliasing  -Wno-uninitialized")
+     SET(CMAKE_C_FLAGS_DEBUG          "-g -O0 ${COMMON_C_FLAGS}")
+     SET(CMAKE_C_FLAGS_RELEASE        "-O3    ${COMMON_C_FLAGS}") # -g
+   ENDIF()
+   IF(CMAKE_COMPILER_IS_GNUCXX)
+     SET(COMMON_CXX_FLAGS               "-fno-omit-frame-pointer -fno-strict-aliasing -Wno-uninitialized")
+     SET(CMAKE_CXX_FLAGS_DEBUG          "-g -O0 ${COMMON_CXX_FLAGS}")
+     SET(CMAKE_CXX_FLAGS_RELEASE        "-O3    ${COMMON_CXX_FLAGS}") # -g
```

Comment:
may as well include `-g` in the common consistent with other platforms. Production back traces without symbols in a backtrace are really hard to debug.

Reply:
OK.